test/integration/create-community-controller.spec.ts

```diff
@@ -268,6 +269,32 @@ test('Create Community Controller', async function ({ components, stubComponents
               expect(await response.json()).toMatchObject({ message: 'Failed to fetch names' })
             })
           })
+
+          describe('and the community privacy is private', () => {
+            let validBodyWithPrivatePrivacy = {
+              ...validBody,
+              privacy: CommunityPrivacyEnum.Private
+            }
```

Comment:
To follow our testing standards closely, would you mind defining this in a `beforeEach`?
If `validBody` was defined inside a `beforeEach`, you could have just done:
```
beforeEach(() => {
  validBody.privacy = CommunityPrivacyEnum.Private
})
```
Which removes the necessity of creating a new variable for it.

Reply:
Updated to make it that way! thx